readme.rst

```diff
@@ -454,6 +454,37 @@ Validate links within the documentation:
 
    make linkcheck
 
+Style guide linting
+^^^^^^^^^^^^^^^^^^^
+
+Check documentation against the `Vale documentation linter configured with the current style guide <https://github.com/canonical/praecepta>`_.
```

Comment:
The spelling check is failing on "linter", so you'll need to add that to the wordlist.